_api-reference/list/index.md

```diff
@@ -0,0 +1,175 @@
+
+
+### Get verbose output
+
+To query indexes and their stats with a verbose output that includes all column headings in the response, use the `v` query parameter, as shown in the following example:
+
+#### Request
+
+```json
+GET _list/indices?v
+```
+{% include copy-curl.html %}
+
+#### Response
+
+```json
+health status index           uuid    pri rep  docs.count  docs.deleted
+green  open   .kibana_1 - - - -              
+yellow open    sample-index-1 - - - -
+next_token null
+```
+
+
+### Get all available headers
+
+To see all the available headers, use the `help` parameter with the following syntax:
+
+```json
+GET _list/<operation_name>?help
+```
+{% include copy-curl.html %}
+
+#### Request
+
+The following example List Indices operation returns all the available headers:
+
+```json
+GET _list/indices?help
+```
+{% include copy-curl.html %}
+
+#### Response
+
+The following example shows the indexes and their health status in a table:
+
+```json
+health     | h                              | current health status
+status     | s                              | open/close status
+index      | i,idx                          | index name
+uuid       | id,uuid                        | index uuid
+pri        | p,shards.primary,shardsPrimary | number of primary shards
+rep        | r,shards.replica,shardsReplica | number of replica shards
+docs.count | dc,docsCount                   | available docs
+```
+
+### Get a subset of headers
+
+To limit the output to a subset of headers, use the `h` parameter with the following syntax:
+
+```json
+GET _list/<operation_name>?h=<header_name_1>,<header_name_2>&v
+```
+{% include copy-curl.html %}
+
+For any operation, you can find out what headers are available using the `help` parameter, and then use the `h` parameter to limit the output to only the headers that you care about. 
+
+#### Request
+
+The following example limits indexes in the response to only the index name and health:
+
+```json
+GET _list/indices?h=health,index
+```
+{% include copy-curl.html %}
+
+### Response
+
+```json
+green  .kibana_1
+yellow sample-index-1
+next_token null
+```
+
+
+### Sort by a header
+
+To sort the output in a single page by a header, use the `s` parameter with the following syntax:
+
+```json
+GET _list/<operation_name>?s=<header_name_1>,<header_name_2>
+```
+{% include copy-curl.html %}
+
+#### Request
+
+The following example request sorts indexes by health index name:
+
+```json
+GET _list/indices?s=h,i
+```
+{% include copy-curl.html %}
+
+#### Response
+
+```json
+green sample-index-2
+yellow sample-index-1
+next_token null
+```
+
+### Retrieve data in JSON
+
+By default, List APIs return data in a `text/plain` format. Other supported formats are [YAML](https://yaml.org/), [CBOR](https://cbor.io/), and [Smile](https://github.com/FasterXML/smile-format-specification).
+
+
+To retrieve data in JSON format, use the `format=json` parameter with the following syntax:
```

Comment:
No syntax follows this. Should this just end in a period if it refers to the following sections?

Reply:
Yes.